src/main/java/module/lineup/Lineup.java

```diff
@@ -101,7 +97,15 @@ public void setPlayers(List<MatchLineupPosition> matchLineupPositions) {
 		}
 	}
 
-	private static class Settings {
+	public long getRatingRevision() {
+		return ratingRevision;
+	}
+
+    public void setSubstitutionsUpdated() {
+		ratingRevision++;
```

Comment:
What is `ratingRevision` for?

Reply:
Some of the new rating caches register the lineup's rating revision when they calculate. Changing of skill values and other properties, that changes the results of those calculations increments the revision number. Then the caches are resetted.

Reply:
comment added